lib/ui/widget/discounts/discount_view.dart

```diff
@@ -61,7 +72,35 @@ class DiscountView extends StatelessWidget {
                                     horizontal: 90.0),
                                 width: double.infinity,
                                 height: 80,
-                                color: Colors.grey[700],
+                                color: Colors.grey[500],
```

Comment:
Should use Theme.of(context). ... etc, here do not use Color Directly we have discussed this.